programs/marinade-finance/src/instructions/liq_pool/add_liquidity.rs

```diff
@@ -149,6 +153,21 @@ impl<'info> AddLiquidity<'info> {
         )?;
         self.state.liq_pool.on_lp_mint(shares_for_user);
 
+        self.mint_to.reload()?;
+        self.lp_mint.reload()?;
+        emit!(AddLiquidityEvent {
```

Comment:
1) we don't use mSOL price in this instruction. AddLiquidity only adds SOL to the swap pool, mSOL price is not relevant.
Let's remove that and we should add current liquid-unstake fee, to check the relation between curren fee and deposits
2) isn't it better to add to the event: previous_lp_supply instead of new_lp_supply (we avoid a reload()?

Reply:
1) we don't use mSOL price in this instruction. AddLiquidity only adds SOL to the swap pool, mSOL price is not relevant.
2) Let's remove that and we should add current liquid-unstake fee, to check the relation between curren fee and deposits
3) isn't it better to add to the event: previous_lp_supply instead of new_lp_supply (we avoid a reload()?